spec/features/user_views_and_edits_entries_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+feature "User views and edits entries" do
+  scenario "successfully" do
+    user = create(:user)
+    feed = create(:feed, user: user)
+    entry = create(:entry, feed: feed)
+
+    visit root_path(as: user)
+    click_on(feed.name)
+
+    expect(page).to have_feed_entry(entry.headline)
+
+    click_on(entry.headline)
+
+    fill_form(:entry, {
+      headline: "This is an edited headline",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.